getting_started/step_by_step/your_first_game.rst

```diff
@@ -666,8 +666,11 @@ This completes the `Mob` scene.
 Main scene
 ----------
 
-Now it's time to bring it all together. Create a new scene and add a
-:ref:`Node <class_Node>` named ``Main``. Click the "Instance" button and select your
+Now it's time to bring it all together. Create a new scene, click on
+the other node button and create a
```

Comment:
>  click on the other node button

This is the third scene we've created so far, so this seems unnecessary. At least title it consistently: "Other Node" button